QRCoder/PayloadGenerator.cs

```diff
@@ -420,11 +421,12 @@ public class ContactData : Payload
             /// <param name="country">Country</param>
             /// <param name="addressOrder">The address order format to use</param>
             /// <param name="note">Memo text / notes</param>            
-            public ContactData(ContactOutputType outputType, string firstname, string lastname, string nickname = null, string phone = null, string mobilePhone = null, string workPhone = null, string email = null, DateTime? birthday = null, string website = null, string street = null, string houseNumber = null, string city = null, string zipCode = null, string country = null, string note = null, string stateRegion = null, AddressOrder addressOrder = AddressOrder.Default)
+            public ContactData(ContactOutputType outputType, string firstname, string lastname, string nickname = null, string org = null, string phone = null, string mobilePhone = null, string workPhone = null, string email = null, DateTime? birthday = null, string website = null, string street = null, string houseNumber = null, string city = null, string zipCode = null, string country = null, string note = null, string stateRegion = null, AddressOrder addressOrder = AddressOrder.Default)
```

Comment:
Can you add the new parameter "org" as last parameter to make it "non-breaking"? Otherwise it may break existing solutions. E.g. if someone calls the function without named parameters...
`ContactData(outputType, "Michael", "Meyers", "Mike", "0900 123456");`
...the phonenumber would be treated as org after merging your change.

Reply:
Good catch. Changed the ctor and all tests to be non-breaking.